Added a formal group policy document (#14)
UniversityOfSaskatchewanCMPT371/term-project-fall2019-team-2: documents/policy/policy.tex

    pdfkeywords={TeX, LaTeX, graphics, images, policy}, % list of keywords
    colorlinks=true,       % false: boxed links; true: colored links
    linkcolor=blue,       % color of internal links
    citecolor=black,       % color of links to bibliography
    filecolor=black,        % color of file links
    urlcolor=purple,        % color of external links
    linktoc=page            % only page is linked
}%
\def\myversion{1.0 }
\date{}
%\title{%

%}
\usepackage{hyperref}
\begin{document}

\graphicspath{ {images/} }

\begin{flushright}
    \rule{16cm}{5pt}\vskip1cm
    \begin{bfseries}
        \Huge{TEAM 2 POLICY}\\
        \vspace{1.9cm}
        for\\
        \vspace{1.9cm}
        CMPT371\\
        \vspace{1.9cm}
        \LARGE{Version \myversion approved}\\
        \vspace{1.9cm}
        Prepared by Team 2\\
        \vspace{1.9cm}
        University of Saskatchewan\\
        \vspace{1.9cm}
        \today\\
    \end{bfseries}
\end{flushright}

\tableofcontents

\chapter{Policy}

\section{Members}

\begin{table}[h!]
\centering
\begin{tabular}{ |p{2cm}||p{8cm}|  }
    \hline
    \multicolumn{2}{|c|}{Team Roles} \\
    \hline
    Member & Roles\\
    \hline

>     Amanda & Project Lead\\

Author: Please review these to ensure all roles are correct. I guessed for a few of theme.

Reviewer: Should be:
Evan - Project Lead
Eileen - Test Lead 
Amanda - Developer

        \vspace{1.9cm}
        for\\
        \vspace{1.9cm}
        CMPT371\\
        \vspace{1.9cm}
        \LARGE{Version \myversion approved}\\
        \vspace{1.9cm}
        Prepared by Team 2\\
        \vspace{1.9cm}
        University of Saskatchewan\\
        \vspace{1.9cm}
        \today\\
    \end{bfseries}
\end{flushright}

\tableofcontents

\chapter{Policy}

\section{Members}

\begin{table}[h!]
\centering
\begin{tabular}{ |p{2cm}||p{8cm}|  }
    \hline
    \multicolumn{2}{|c|}{Team Roles} \\
    \hline
    Member & Roles\\
    \hline
    Amanda & Project Lead\\
    Mesa & Design Lead\\
    Evan & Test Lead\\
    Braunson & Build Master\\
    Kevin & Risk Officer, Development Team, Test Team\\
    Clinton & Test Team\\
    Anurag & Development Team, Test Team\\
    Eileen & Development Team\\
    Camille & Development Team\\
    \hline
\end{tabular}
\end{table}

\section{Roles and Responsibilities}

\subsection{Project Lead}

The project lead will be responsible for organizing meetings,
meeting with stakeholders, establishing priority, coordinating
between accountable positions, and other administrative work. 
Generally, the project lead will not directly code.

> In cases of disagreements the project lead has executive decision making power.

Reviewer: Establishing priority of what? Should probably clarify that a bit. Also, I'm assuming "administrative work" includes emailing the stakeholders, taking meeting minutes, etc. (Just wondering how much detail we want with that).

Author: Adjusted the priority line.

And yea I note in this paragraph they are responsible for organizing meetings (agenda/minutes), and I mention meeting with stakeholders. Other administrative work could be emailing them right.

The design lead is in charge of development. Their goal is
to complete the priorities set out by the stakeholder, organize
the development team to ensure cohesion, and selecting technologies
for use in the project. They should also ensure code quality and
work closely with the test lead and build master.

\subsection{Test Lead}

The test lead has to ensure there is sufficient test coverage
that adequately tests use cases of the stakeholder. They will
have oversight over the whole test team and is in charge of 
test plans.

\subsection{Build Master}

The build master ensures the build pipeline is maintained with
a server to deploy as staging and production environments. The
pipeline managed by the build master should include automating
tests, notifying group members of build issues, and ensuring
quality code is checked into the project.

\subsection{Risk Officer}

The risk officer is a rotating, part-time role which evaluates
risk continuously during project development. They are responsible
for documenting mitigation strategies and proactively taking
steps to reduce risks to the project. 

\section{Meetings}

Meetings and class room attendance is considered mandatory. 

> If an individual cannot attend a meeting they are to notify the project lead ahead of time and read the meeting minutes after. If more than three meetings are missed in the course of the term that person must talk to Amanda, if any more meetings are missed the team will talk to Dr. Osgood.

Reviewer: ```suggestion
If an individual cannot attend a meeting they are to notify the project lead ahead of time and must read the meeting minutes afterwards/when they are able to/at their earliest opportunity. If more than three meetings are missed in the course of the term that person must talk to Amanda, if any more meetings are missed, the team will talk to Dr. Osgood.
```

> Ten minute meetings before class will be used as an informal stand up.

Reviewer: ```suggestion
The first ten minutes of class will be used as an informal stand up.
```

> If an individual cannot attend a meeting they are to notify the project lead ahead of time and must read the meeting minutes afterwards/when they are able to/at their earliest opportunity. If more than three meetings are missed in the course of the term that person must talk to Amanda, if any more meetings are missed, the team will talk to Dr. Osgood.

Reviewer: ```suggestion
If an individual cannot attend a meeting they are to notify the project lead ahead of time and must read the meeting minutes afterwards. If more than three meetings are missed in the course of the term that person must talk to Amanda, if any more meetings are missed, the team will talk to Dr. Osgood.
```
